Update link to playlist (#67)
spinalcordmri/spinalcordmri.github.io: _posts/2020-01-20-workshop.md

### Live streaming

> For those who could not attend, the workshop and SCT course will be live streamed and recorded. You will need to install [Zoom](https://zoom.us/). The workshop will be streamed [here](https://zoom.us/j/695903361), and the SCT course will be streamed [here](https://zoom.us/j/606682393). Recordings of the workshop from both days can be found [here](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql). Recording of the SCT course can be found [here](https://youtu.be/whbtjYNtHko).

Reviewer: @alexfoias as mentioned in the todo list pls create separate section for recordings

Author: The current link point to the playlist workshops (where all the workshops are listed). You would like a separate playlist called Recordings and migrate the spinal cord workshop 7 videos to it ?

Author: @jcohenadad The [current link](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql) points to the playlist workshops (where all the workshops are listed). You would like a separate playlist called Recordings and migrate the spinal cord workshop 7 videos to it ?

Author: or do you mean section in the actual post that points to the youtube playlist?

> For those who could not attend, the workshop and SCT course will be live streamed and recorded. You will need to install [Zoom](https://zoom.us/). The workshop will be streamed [here](https://zoom.us/j/695903361), and the SCT course will be streamed [here](https://zoom.us/j/606682393). Recordings of the workshop from both days can be found [here](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql). Recording of the SCT course can be found [here](https://youtu.be/whbtjYNtHko).

Reviewer: ```suggestion
For those who could not attend, the workshop and SCT course will be live streamed and recorded. You will need to install [Zoom](https://zoom.us/). The workshop will be streamed [here](https://zoom.us/j/695903361), and the SCT course will be streamed [here](https://zoom.us/j/606682393). 

### Recordings
Recordings of the full workshop is available [here](https://www.youtube.com/playlist?list=PLJ5-Fnq9XpaW7hLYbQCZaxGAp_iuKUiql). Recordings of the SCT course can be found [here](https://youtu.be/whbtjYNtHko).
```
